README.md

```diff
@@ -2,9 +2,19 @@
 
 A tool for GUI automation using a variety of computer vision and desktop control backends.
 
+## Introduction
+
+In order to do GUI automation you usually need to solve two problems: first, you need to have a way to control and interact with the interface and platform you are automating and second, you need to be able to locate the objects you are interested in on the screen. Guibot helps you do both.
```

Comment:
I think we have had some ambiguity about using the capital letters in the past, frequently making use of `GuiBot` as well. In the last years I have mostly avoided capitalization at all and stayed with `guibot` which is clearly not the best solution. @thomasjfox What capitalization is official?